packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -386,3 +399,63 @@ export async function sideloadAddIn(
     throw err;
   }
 }
+
+function isDotnetInstalled(): boolean {
+  try {
+    // Find the .Net runtimes installed
+    let result = childProcess.execSync("dotnet --list-runtimes");
+    const pattern = /(?<=Microsoft.NETCore.App )[\d.]+/g;
+    const matches = result.toString("utf-8").match(pattern);
+    let foundDotNet = false;
+
+    // Look for version 5 or greater
+    matches?.forEach((match) => {
+      const major: number = parseInt(match.split(".")[0]);
+      foundDotNet = foundDotNet || major >= 5;
+    });
+
+    return foundDotNet;
+  } catch (err) {
+    return false;
+  }
+}
+
+async function convertJsonToXmlManifest(manifestPath: string): Promise<string> {
+  return new Promise((resolve, reject) => {
+    if (manifestPath.endsWith(".json") && fs.existsSync(manifestPath)) {
+      console.log("Converting json to back compat xml");
+      const newManifestPath = path.join(process.env.TEMP as string, "manifest.xml");
+      const command = `.\\lib\\DevXTool.exe ${manifestPath} ${newManifestPath}`;
+
+      childProcess.exec(command, (error, stdout) => {
+        if (error) {
+          console.log(`Error converting file:\n ${stdout}\n ${error}`);
+          reject("");
+        } else {
+          console.log(`Successfully converted manifest to xml:\n ${stdout}`);
+          stripBom(newManifestPath);
+          resolve(newManifestPath);
+        }
+      });
+    } else {
+      reject(new Error(`The file '${manifestPath}' is not valid`));
+    }
+  });
+}
+
+export default function stripBom(manifestPath: string) {
```

Comment:
What does `Bom` mean? Why do we need this function?

Reply:
https://docs.microsoft.com/en-us/globalization/encoding/byte-order-mark

It was getting in the way of outlook loading the add-in because the resulting xml had the BOM.  utf-8 doesn't need it so I removed it.